seedrandom.js

```diff
@@ -35,7 +35,7 @@ var global = this,
     significance = math.pow(2, digits),
     overflow = significance * 2,
     mask = width - 1,
-    nodecrypto;         // node.js crypto module, initialized at the bottom.
```

Comment:
@davidbau another options is we could make a browser distribution. Would introduce a build step in your process though. for example:

```bash
browserify --standalone -o seedrandom.browser.js
```
That will produce a UMD bundle that works in all module environments and will expose it as a global if none are defined.

Actually this might be nice because you could just use `require` like a normal node module  throughout the code, and then this standalone build would produce a browser friendly file.

thoughts?